editoast/osm_to_railjson/README.md

```diff
@@ -0,0 +1,17 @@
+# Converter from OSM to raijson
+
+## Usage
+
+Example for Germany:
+
+1. Download OSM's Germany data (~4Gb `germany-latest.osm.pbf`) from
+    https://download.geofabrik.de/europe/germany.html
+2. Launch osrd conversion (release build of editoast and conversion can be long):
+    ```sh
+    cd ../../editoast
+    cargo run -r -- osm-to-railjson <path/to/germany-latest.osm.pbf> <path/to/germany_railjson.json>
+    ```
+3. Load railjson using the script (or through OSRD's web interface):
+    ```sh
+    ../../scripts/load-railjson-infra.sh "germany" <path/to/germany_railjson.json>
```

Comment:
Since we used `cargo` for the main command, I feel like it's safe to suggest instead
```suggestion
    cargo run -r -- infra import-railjson "germany" <path/to/germany_railjson.json>
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8825/commits/4722e8406c7d1ba4955e09509372e73c27fad703 (kept the mention of the script in alternatives)